### ISSUE TITLE
Update pypiscout required version to v2.0.0

### ISSUE THREAD

**Issue by KGergo88** [bmwcarit/Emma#8]
This version is needed because this is the first one to have the Logger class.

**Comment by holzkohlengrill** [comment]
This issue is dependent on issue #5 .
Previous versions do not need this requirement.

**Comment by holzkohlengrill** [comment]
What is still to do in order to solve this issue?

**Comment by holzkohlengrill** [comment]
Currently not all pypiscout calls (esp. Emma-vis) are used with the SCout Logger implementation.

**Comment by KGergo88** [comment]
> What is still to do in order to solve this issue?

Not much, will be closed after merging the #5.

**Comment by KGergo88** [comment]
> Currently not all pypiscout calls (esp. Emma-vis) are used with the SCout Logger implementation.

True, since Deltas and the Vis is not part of #5 these are not planned to be changed at the moment.
@holzkohlengrill could solve this as part of a new issue.
